bokehjs/src/coffee/api/typings/plotting.d.ts

```diff
@@ -1,6 +1,8 @@
 declare namespace Bokeh.Plotting {
     function show(figure: Figure, target?: HTMLElement): void;
 
+    function color(r: number, g: number, b: number): string;
+
```

Comment:
Do we have this function on the Python side? If not, maybe `rgb2hex` would be more clear.